Tests/GitHubReferences.Tests.ps1

```diff
@@ -0,0 +1,200 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
This file has been refactored a bit, mostly with respect to usage of names like "heads/master" which have been replaced with variables